<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>152022099 Cikal GS</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Praktikum PCD - (152022099) Cikal Gemintang Seya</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="originalLabel">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>16</y>
      <width>289</width>
      <height>257</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>288</y>
      <width>145</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Muat Citra</string>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>16</y>
      <width>289</width>
      <height>257</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="resultButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>288</y>
      <width>145</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply</string>
    </property>
   </widget>
   <widget class="QComboBox" name="filterCombo">
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>288</y>
      <width>273</width>
      <height>32</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Grayscale</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Brightness</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Contrast</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Stretching</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Negative</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Binary</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>192</x>
      <y>336</y>
      <width>273</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="pageStep">
     <number>0</number>
    </property>
    <property name="value">
     <number>80</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="swapButton">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>328</y>
      <width>145</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Tukar dari Hasil</string>
    </property>
   </widget>
   <widget class="QPushButton" name="popupButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>328</y>
      <width>145</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Besarkan Citra</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>662</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="hgMenu">
    <property name="title">
     <string>Histogram</string>
    </property>
    <addaction name="hgGrayscale"/>
    <addaction name="hgRGB"/>
    <addaction name="hgEqualization"/>
   </widget>
   <widget class="QMenu" name="tfMenu">
    <property name="title">
     <string>Transformasi</string>
    </property>
    <addaction name="tfTranslasi"/>
    <addaction name="tfRotasi"/>
    <addaction name="tfSkala"/>
    <addaction name="tfCrop"/>
   </widget>
   <widget class="QMenu" name="opMenu">
    <property name="title">
     <string>Operasi Pixel</string>
    </property>
    <addaction name="opAdd"/>
    <addaction name="opSubtract"/>
    <addaction name="separator"/>
    <addaction name="opAND"/>
    <addaction name="opOR"/>
    <addaction name="opXOR"/>
   </widget>
   <widget class="QMenu" name="kvMenu">
    <property name="title">
     <string>Konvolusi</string>
    </property>
    <widget class="QMenu" name="kvSmoothing">
     <property name="title">
      <string>Smoothing</string>
     </property>
     <addaction name="kvMean"/>
     <addaction name="kvGaussian"/>
    </widget>
    <widget class="QMenu" name="kvSharpening">
     <property name="title">
      <string>Sharpening</string>
     </property>
     <addaction name="kvSharpI"/>
     <addaction name="kvSharpII"/>
     <addaction name="kvSharpIII"/>
     <addaction name="kvSharpIV"/>
     <addaction name="kvSharpV"/>
     <addaction name="kvSharpVI"/>
    </widget>
    <addaction name="kvSmoothing"/>
    <addaction name="kvSharpening"/>
    <addaction name="kvMedian"/>
    <addaction name="kvMax"/>
   </widget>
   <addaction name="hgMenu"/>
   <addaction name="tfMenu"/>
   <addaction name="opMenu"/>
   <addaction name="kvMenu"/>
  </widget>
  <action name="hgGrayscale">
   <property name="text">
    <string>Grayscale</string>
   </property>
  </action>
  <action name="hgRGB">
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="hgEqualization">
   <property name="text">
    <string>Equalization</string>
   </property>
  </action>
  <action name="tfSkala">
   <property name="text">
    <string>Skala</string>
   </property>
  </action>
  <action name="tfTranslasi">
   <property name="text">
    <string>Translasi</string>
   </property>
  </action>
  <action name="tfCrop">
   <property name="text">
    <string>Crop</string>
   </property>
  </action>
  <action name="tfRotasi">
   <property name="text">
    <string>Rotasi</string>
   </property>
  </action>
  <action name="opAdd">
   <property name="text">
    <string>Add</string>
   </property>
  </action>
  <action name="opSubtract">
   <property name="text">
    <string>Subtract</string>
   </property>
  </action>
  <action name="opAND">
   <property name="text">
    <string>AND</string>
   </property>
  </action>
  <action name="opOR">
   <property name="text">
    <string>OR</string>
   </property>
  </action>
  <action name="opXOR">
   <property name="text">
    <string>XOR</string>
   </property>
  </action>
  <action name="kvMean">
   <property name="text">
    <string>Mean Filter</string>
   </property>
  </action>
  <action name="kvGaussian">
   <property name="text">
    <string>Gaussian Filter</string>
   </property>
  </action>
  <action name="kvSharpI">
   <property name="text">
    <string>i</string>
   </property>
  </action>
  <action name="kvSharpII">
   <property name="text">
    <string>ii</string>
   </property>
  </action>
  <action name="kvSharpIII">
   <property name="text">
    <string>iii</string>
   </property>
  </action>
  <action name="kvSharpIV">
   <property name="text">
    <string>iv</string>
   </property>
  </action>
  <action name="kvSharpV">
   <property name="text">
    <string>v</string>
   </property>
  </action>
  <action name="kvSharpVI">
   <property name="text">
    <string>vi</string>
   </property>
  </action>
  <action name="kvMedian">
   <property name="text">
    <string>Median Filter</string>
   </property>
  </action>
  <action name="kvMax">
   <property name="text">
    <string>Max Filter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
